shuup/front/apps/carousel/locale/fi_FI/LC_MESSAGES/django.po

```diff
@@ -159,6 +165,18 @@ msgstr "Aseta päivä ja aika, milloin tämän karusellikohteen pitäisi olla n
 msgid "available to"
 msgstr "saatavilla asti"
 
+msgid "Inactive dot border color"
+msgstr "Ei aktiivisen pisteen reunan väri"
+
+msgid "Customize the dot border color when slide is not active."
+msgstr "Aseta pisteen reunan väri kun liuku ei ole aktiivinen."
```

Comment:
This "Liuku" has always been bad. The Google translate suggests it but it is correct when used in "I'm sliding downhill" so I was thinking should we use `bannerikuva` on all these "slide" translations?